android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -173,7 +175,9 @@ class CustomTabsToolbarFeature(
             context.getString(R.string.mozac_feature_customtabs_exit_button),
         ) {
             emitCloseFact()
-            useCases.remove(tab.id)
+            session ?. let {
```

Comment:
```suggestion
            session?.let {
```
The spacing doesn't quite follow what we typically do - so requesting changes here for conistency.